src/Shared/Recruit.Vacancies.Client/Infrastructure/ReferenceData/BankHolidays/BankHolidayUpdateService.cs

```diff
@@ -1,48 +1,67 @@
 ﻿using System;
 using System.Linq;
 using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Application.Providers;
 using Esfa.Recruit.Vacancies.Client.Application.Services.ReferenceData;
-using Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData;
-using Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.BankHolidays;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using RestSharp;
 
-namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Services
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.BankHolidays
 {
     public class BankHolidayUpdateService : IBankHolidayUpdateService
     {
         private readonly BankHolidayConfiguration _config;
         private readonly IReferenceDataWriter _referenceDataWriter;
         private readonly ILogger<BankHolidayUpdateService> _logger;
+        private readonly IBankHolidayProvider _bankHolidayProvider;
```

Comment:
I'd remove this reference to the `IBankHolidayProvider` and just add a reference to `IReferenceDataReader ` and get the bank holidays directly.